components/cronie.yml

```diff
@@ -6,12 +6,15 @@ packages:
 - cronie
 rules:
 - disable_anacron
+- file_at_allow_exists
+- file_at_deny_exists
```

Comment:
I think this line should be removed since the new rule `file_at_deny_exists` was removed from the PR.

Reply:
Removed